string name                         # object's name
uint8 pos_id                        # position id
geometry_msgs/Vector3 position      # grasping position
--- 
